scripts/Muon/GUI/Common/calculate_pair_and_group.py

```diff
@@ -26,6 +26,15 @@ def calculate_pair_data(context, pair_name, run):
 
     return pair_data
 
+def estimate_group_asymmetry_data(context, group_name, run):
+    processed_data = _run_pre_processing(context, run)
+
+    params = _get_MuonGroupingAsymmetry_parameters(context, group_name, run)
+    params["InputWorkspace"] = processed_data
+    pair_data = algorithm_utils.run_MuonGroupingAsymmetry(params)
```

Comment:
I think we should change the variable name, as far as I can tell this is not a muon pair